crates/holochain/tests/tests/conductor_services/mod.rs

```diff
@@ -9,7 +9,7 @@ async fn initialize_dpki() {
     holochain_trace::test_run();
 
     let mut config = SweetConductorConfig::standard();
-    config.dpki = DpkiConfig::production(None);
+    config.dpki = DpkiConfig::testing();
```

Comment:
Was this using production config in a test? Or was the `None` relevant here somehow?